linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/interceptor/impl/CommentInterceptor.scala

```diff
@@ -70,8 +70,32 @@ trait CommentHelper {
 object SQLCommentHelper extends CommentHelper {
   override val commentPattern: Regex = """\s*--.+\s*""".r.unanchored
   private val comment = "(?ms)('(?:''|[^'])*')|--.*?$|/\\*.*?\\*/|#.*?$|"
+  private val comment_sem = "(?i)(comment)\\s+'([^']*)'"
   private val logger: Logger = LoggerFactory.getLogger(getClass)
 
+  def replaceComment(code: String): String = {
+    try {
+      val pattern = Pattern.compile(comment_sem)
+      val matcher = pattern.matcher(code)
+      val sb = new StringBuffer
+      while (matcher.find()) {
+        val commentKeyword = matcher.group(1)
+        val comment = matcher.group(2)
+        val escapedComment = comment.replaceAll(";", "\\\\\\\\;")
```

Comment:
batter to add some comments to explain  for using "\\\\\\\\" (eight  \)

Reply:
Please review again.Thanks!

Reply:
Please review again.Thanks!